apps/base-docs/docs/building-with-base/guides/hyperframes.md

```diff
@@ -0,0 +1,327 @@
+---
+title: 'HyperFrames'
+description: Learn how to make cross-linked hyperframes in an organized manner.
+hide_table_of_contents: false
+---
+
+[Frames] on [Farcaster] are getting more complex. Developers are now building interactions that require a handful, or even dozens of frames in response to various user states, inputs, and actions. HyperFrames are a system to organize navigation for large numbers of frames, using [OnchainKit]. In this tutorial, we'll use making the navigation for an [old-school adventure game] fully in a frame. You can use this same technique for many other intents with your Frames, such as games, stores, customized mints, etc.
+
+:::caution
+
+Frames are brand new and tools are evolving quickly. Check the links above for changelogs!
```

Comment:
Consider including the change log links directly within this admonition. There's 3 different links above.